recipes/crashpad/all/conanfile.py

```diff
@@ -0,0 +1,275 @@
+        if self.settings.compiler == "clang":
+            if self.settings.compiler.get_safe("libcxx"):
+                stdlib = {
+                    "libstdc++11": "libstdc++",
+                }.get(str(self.settings.compiler.libcxx), str(self.settings.compiler.libcxx))
+                extra_cflags_cc.append("-stdlib={}".format(stdlib))
+                extra_ldflags.append("-stdlib={}".format(stdlib))
+        gn_args = [
+            "host_os=\\\"{}\\\"".format(self._gn_os),
+            "host_cpu=\\\"{}\\\"".format(self._gn_arch),
+            "is_debug={}".format(str(self.settings.build_type == "Debug").lower()),
+            "crashpad_http_transport_impl=\\\"{}\\\"".format(self._http_transport_impl),
+            "crashpad_use_boringssl_for_http_transport_socket={}".format(str(self.options.get_safe("with_tls", False) != False).lower()),
+            "extra_cflags=\\\"{}\\\"".format(" ".join(extra_cflags)),
+            "extra_cflags_c=\\\"{}\\\"".format(" ".join(extra_cflags_c)),
+            "extra_cflags_cc=\\\"{}\\\"".format(" ".join(extra_cflags_cc)),
+            "extra_ldflags=\\\"{}\\\"".format(" ".join(extra_ldflags)),
+        ]
+        with tools.chdir(self._source_subfolder):
+            with self._build_context():
+                self.run("gn gen out/Default --args=\"{}\"".format(" ".join(gn_args)), run_environment=True)
+                for target in ("client", "minidump", "crashpad_handler", "snapshot"):
+                    # FIXME: Remove verbose once everything is working hunky dory
+                    self.run("ninja -C out/Default {target} -j{parallel}".format(
+                        target=target,
+                        parallel=tools.cpu_count()), run_environment=True)
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "client"), dst=os.path.join("include", "client"))
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "util"), dst=os.path.join("include", "util"))
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "third_party", "mini_chromium", "mini_chromium", "base"), dst=os.path.join("include", "base"))
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "third_party", "mini_chromium", "mini_chromium", "build"), dst=os.path.join("include", "build"))
+        self.copy("*.h", src=os.path.join(self._source_subfolder, "out", "Default", "gen", "build"), dst=os.path.join("include", "build"))
+
+        self.copy("*.a", src=os.path.join(self._source_subfolder, "out", "Default"), dst="lib", keep_path=False)
+        self.copy("crashpad_handler", src=os.path.join(self._source_subfolder, "out", "Default"), dst="bin", keep_path=False)
+        self.copy("crashpad_handler.exe", src=os.path.join(self._source_subfolder, "out", "Default"), dst="bin", keep_path=False)
```

Comment:
On Windows there seems to be two executables provided.  (missing .com)

https://chromium.googlesource.com/crashpad/crashpad/+/master/doc/overview_design.md
> On Windows, this executable is built by default as a Windows GUI app, so no console will appear in normal usage. This is the version that will typically be used. A second copy is also made with a .com extension, rather than .exe. In this second copy, the PE header is modified to indicate that it’s a console app. This is useful because the .com is found in the path before the .exe, so when run normally from a shell using only the basename (without an explicit .com or .exe extension), the .com console version will be chosen, and so stdio will be hooked up as expected to the parent console so that logging output will be visible.

Reply:
It works on Python 3.9.5 on Linux.
But an error is thrown when building on Windows.
Maybe it can be patched..

Reply:
I added it. It's called `crashpad_handler_com.com`.

Reply:
How come it's called `crashpad_handler_com.com` instead of just `crashpad_handler.com` as according to the documentation?